src/OpenSage.Game/Content/ContentManager.cs

```diff
@@ -1,9 +1,9 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Linq;
-using OpenSage.Audio;
```

Comment:
It just removes an unused `using` statement. 😄 The audio system is just as non-existing as it was before.

Reply:
Sure, but it looks like it's a placeholder then, for something that's surely going to be used. Ofcourse this just may be a reference, not the audio it self so in this case it may not be needed in that specific file.